websockets/1.0/modules/ROOT/pages/websockets-connector-examples.adoc

```diff
@@ -239,14 +234,74 @@ The received `QuoteSnapshots` should be transformed to CSV format and appended t
 			</when>
 		</choice>
 	</flow>
+
+</mule>
+----
+
+== Quote Aggregator App
+
+The following Mule app connects to the feed endpoint of the previous Quote Producer App and receives all of the quotes. Then, the Quote Aggregator App splits the quotes by ticker and places them in time-based aggregators that occur every 5 seconds.
+
+.Quote Aggregator App flow
+image::websocket-example-2.png[Quote Aggregator App flow in Anypoint Studio Canvas]
+
+When the aggregations occur, the aggregators output an `Array<Quote>` to reduce to a single `QuoteSnapshot` type.
+
+.QuoteSnapshot Type example:
+[source,json,linenums]
 ----
+{
+   "ticker": "CRM"
+   "price": "157.54 USD"
+}
+----
+
+* The reduction occurs by picking the `Quote` with the greatest timestamp and transforming it per the example.
+* The produced snapshots broadcast to a dynamic list of subscribers.
+* Subscriptions are done through the `+wss://localhost:8082/quotes+` endpoint that the Quote Aggregator App exposes.
+* Clients can connect to the previous endpoint by using a query parameter to indicate what stock quotes to follow, for example: `+wss://localhost:8082/quotes?ticker=CRM&ticker=MELI+`
 
-== Quote Aggregator Implementation
+To generate the corresponding keystores and truststores:
+
+. Open your terminal.
```

Comment:
Weird to have this step and then have the user do something in Studio. I suggest adding this to the navigate step:

Open terminal and run the following command to generate the keystores. (Does this generate 1 keystore or multiple ones? If one, keystores > keystore)